bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java

```diff
@@ -142,19 +146,34 @@ public Response getInterpreter(
     public Response interpret(
             @HeaderParam(HttpHeaders.ACCEPT_LANGUAGE) @Parameter(description = "language") @Nullable String language,
             @Parameter(description = "text to interpret", required = true) String text,
-            @PathParam("id") @Parameter(description = "interpreter id") String id) {
+            @PathParam("ids") @Parameter(description = "comma separated list of interpreter ids") String ids) {
```

Comment:
Related to my previous comment:
```
@PathParam("id") @Parameter(description = "comma separated list of interpreter ids") String ids) {
```

Reply:
You should be able to implicitly deserialize it to a list (change to `List<String>`).